src/Idris/ParseExpr.hs

```diff
@@ -452,14 +452,19 @@ bracketed' open syn =
             do (FC f start (l, c)) <- getFC
                lchar ')'
                return $ PTrue (spanFC open (FC f start (l, c+1))) TypeOrTerm
-        <|> try (do (ln, lnfc) <- name
-                    colonFC <- lcharFC ':'
-                    lty <- expr syn
-                    starsFC <- reservedOpFC "**"
+        <|> try (do (ln, lnfc, colonFC, lty, starsFC):bnds <- some $ do
+                        (ln, lnfc, colonFC) <- try $ do
+                          (ln, lnfc) <- name
+                          colonFC <- lcharFC ':'
+                          return (ln, lnfc, colonFC)
+                        lty <- expr syn
```

Comment:
Also, I found out that it will parse `Type **` as `Type* *`, I should probably change how `Type*` is parsed.